packages/pn-personafisica-webapp/src/components/Contacts/LegalContactAssociationDialog.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
Remember to write the test suite

Reply:
I have skipped the tests about button clicks for the moment